### DISCUSSION TITLE
Product Icons different in paste compared to evergreen

### DISCUSSION THREAD

**Discussion by pmiller-twilio** [twilio-labs/paste#3628] [General] [answered]
This is copied from a [slack message](https://twilio.slack.com/archives/CFS9UGBSB/p1700575774750819) to the evergreen-paste-migration channel. 

Could someone help me understand something about product icons. I am trying to replace the evergreen sources icon with the paste sources icon. I've attached both here. The first one (in green) is the evergreen icon. Note that the arrow is going out of the circle. In the paste version of the icon (from the paste design docs), the arrow is leading into the circle. Why the change? Am I looking at something wrong?

![image](https://github.com/twilio-labs/paste/assets/114422981/70c4acd2-c58e-4167-a400-280d62c379d3)
![image](https://github.com/twilio-labs/paste/assets/114422981/40599ba4-89df-450a-9ca2-900f93648696)

Note: This also applies to the destinations product icon.

**Comment by serifluous** [comment]
Thanks, @pmiller-twilio ! I created a ticket to fix this—accidentally got Sources and Destinations swapped when we moved the glyphs to Paste. We'll follow up with a link to a PR here soon.

**Reply by serifluous** [reply, marked as answer]
Hey @pmiller-twilio ! The fix will go in when this PR gets merged: #3636 . 

Thanks again for catching the bug.
